test/TestProjects/Models-TypeSpec/Models-TypeSpec.tsp

```diff
@@ -255,39 +255,6 @@ model RoundTripModel extends BaseModel {
   nonRequiredNullableStringList?: string[] | null;
 }
```

Comment:
I saw there still some code left here. Could these move to cadl-ranch as well?

Reply:
fixed